.gitignore

```diff
@@ -29,6 +29,7 @@ downloads/
 eggs/
 .eggs/
 lib/
+!/lib/gitlint/
```

Comment:
I think having a `lib/` directory it legit! I would have removed the `lib/` git-ignore rule instead.

Reply:
we can exclude `lib/` but don't exclude `/lib`. Both have not the same meaning:

- `lib/` will ignore every `lib` directory in the project
- `/lib` will only ignore the `lib` directory at the root of the project and not the other one in the project.

So I think we can keep like I did.